### ISSUE TITLE
Plugin causes job hangs

### ISSUE THREAD

**Issue by vzabauski** [jenkinsci/violation-comments-to-stash-plugin#48]
I have a job that generates lint-compatible xml and violation comments plugin reports those violations to Bitbucket. Sometimes job hangs on "Report violations steps". Console output stops at
```
Archiving artifacts
---
--- Jenkins Violation Comments to Bitbucket Server ---
---
bitbucketServerUrl: https://bitbucket.example.com
projectKey: OI
repoSlug: reponame
pullRequestId: 2079
usernamePasswordCredentialsId: true
username: false
password: false
createSingleFileComments: true
createCommentWithAllSingleFileComments: false
commentOnlyChangedContent: false
commentOnlyChangedContentContext: 0
minSeverity: INFO
keepOldComments: false
LINT with pattern .*\\report.xml
Using username and password from credentials
Pull request: 2079
Workspace: F:\jenkins\jobs\SelfServiceHostConfigValidator\workspace
Found 21 violations from ViolationConfig [pattern=.*\\report.xml, reporter=LINT, parser=LINT].
PR: TPP/reponame/2079 on https://bitbucket.example.com
```
Environment:
Windows Server 2012 R2
Java: 1.8.0.151
I've spotted such behavior on Jenkins 2.73.3, 2.89.2
Plugin version: 1.63-1.65
I've just updated to 1.66 but I'm not sure, if this will help.
Thread dump: [here](https://gist.github.com/vzabauski/8c9983419011ab23f35a52f68927ecf4)

If you need any further information feel free to ask me.

**Comment by tomasbjerre** [comment]
Probably just needs a timeout here:
https://github.com/tomasbjerre/violation-comments-to-bitbucket-server-lib/blob/1.43/src/main/java/se/bjurr/violations/comments/bitbucketserver/lib/client/BitbucketServerInvoker.java#L37

Should be an easy fix.

**Comment by vzabauski** [comment]
Are you planning to include this fix in the next release?

**Comment by tomasbjerre** [comment]
Yes, I'll probably have time to fix it later today =)

**Comment by tomasbjerre** [comment]
Should be fixed in 1.67.

**Comment by vzabauski** [comment]
Thanks a lot!
